docs/develop/01_sdk/02_cookbook/06_upgrading_to_v0_29/01_backwards_compatibility.md

```diff
@@ -3,32 +3,24 @@ id: v29-backwards-compatibility
 title: Backwards Compatibility / Interoperability with Previous Versions
 ---
 
-Beside the breaking code changes we have identified three data format changes which might affect interoperability.
-The didsign.io and w3n.id already accept bare requests for attestation, which are effectively the new Credentials, so nothing should break there.
+Depending on how exactly your application interacts with other applications, the changed data formats might make them incompatible.
```

Comment:
```suggestion
Depending on how exactly your application interacts with other applications, changes to the data formats introduced with v0.29 of the sdk might make your application incompatible with those using older versions.
```

Reply:
should we maybe remind people what this is about ?

Reply:
![grafik](https://user-images.githubusercontent.com/133055/196186592-cdca5026-7ee4-4a69-b8a9-c1753521a7d7.png)
I feel like it would be too much of v0.29, and that from the current version it is clear enough what this is about:
![grafik](https://user-images.githubusercontent.com/133055/196187032-644b08df-5ec9-4e18-8fa1-d436b40af71d.png)